layouts/partials/toc.html

```diff
@@ -0,0 +1,15 @@
+{{/*
+
+  A modified version of the toc.html partial in Docsy.
+
+*/}}
+{{ $page := .Params }}
+{{ if not .Params.notoc -}}
+  {{ with .TableOfContents -}}
+    <hr>
+    <div id="toc">
+        <a id="toc-title" href="#">{{ $page.Title }}</a>
```

Comment:
is the `href="#"` here important? I know that the old ToC code generated one, but I think that might be more by accident than anything else.

Reply:
It allows to go back to the very top of the page, I don't know if it should be considered an important feature :shrug:.

Reply:
If we want to avoid changing the htmltest config, we could set an actual ID to go back to the top.

Reply:
oh, I hadn't realised that was what `href="#"` does. This is fine.